modules/helm-adding-namespace-scoped-helm-chart-repositories.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
```

Comment:
This is a complicated procedure with lots of actions for the user. You wrote it extremely well! :)

Reply:
Thanks @jc-berger :)